src/tests/gfx-gizmos-rotation.edn

```diff
@@ -0,0 +1,109 @@
+                              (Gizmos.Line :A .a :B .z-2 :Color (Float4 0 0 1 1) :Thickness 4)
+
+                              (Float3 1 0 0) >= .xbase
+                              (Float3 0 1 0) >= .ybase
+                              (Float3 0 0 1) >= .zbase
+                              (Float3 -1 1 -1) (Math.Cross (Float3 1 0 0)) >= .norm
+
+                              (Gizmos.Line :A (Float3 0 0 0) :B (Float3 -1 1 -1) :Color (Float4 1 1 1 1) :Thickness 4)
+                              ;; (Gizmos.Disc :Center (Float3 0 0 0) :XBase (Float3 1 0 0) :YBase (Float3 0 1 0) :OuterRadius 1.0 :InnerRadius 0.5 :Color (Float4 0.5 0.5 0.5 1.0))
+
+                              ;; (Gizmos.Disc :Center (Float3 0 0 0) :XBase .xbase :YBase .ybase :OuterRadius 1.0 :InnerRadius 0.5 :Color (Float4 0.2 1.0 0.2 1.0))
+
+                              ;; draws 2 rotation circles around imaginary object
+                              (Gizmos.Circle :Center (Float3 1 1 1) :XBase .xbase :YBase .ybase)
+                              ;; (Gizmos.Circle :Center (Float3 1 1 1) :XBase .zbase :YBase (Float3 -1 1 1) :Color (Float4 0.2 1.0 0.2 1.0) :Thickness 4)
+                              (Gizmos.Circle :Center (Float3 1 1 1) :XBase .xbase :YBase .norm :Color (Float4 0.2 1.0 0.2 1.0) :Thickness 1)
+                              ;; rect on xy plane
+
+                              (Gizmos.Rect :Center (Float3 0 0 0) :XBase .xbase :YBase .norm :Size (Float2 1 1) :Color (Float4 0.2 1.0 0.2 1.0) :Thickness 4)
+
+                              ;; solid rect test on xy plane
+                              ;; (Gizmos.SolidRect :Center (Float3 1 1 0) :XBase .xbase :YBase .ybase :Size (Float2 2 2) :Color (Float4 0 0.5 0 1.0))
+
+                              ;; solid disc test on xy plane
+                              ;; (Gizmos.Disc :Center (Float3 1 1 0.5) :XBase .xbase :YBase .ybase :OuterRadius 5.0 :InnerRadius 3.0 :Color (Float4 0 0 0.5 1.0))
+
+                              ;; imaginary object (box)
+                              (Gizmos.Box :Center (Float3 1 1 1) :Size (Float3 0.5 0.2 0.3) :Color (Float4 0.4 0.4 1.0 1.0) :Thickness 4)
+
+                              ;; dot at top right of xy rect
+                              (Gizmos.Point :Center (Float3 1.4 1.4 1) :Color (Float4 0.4 0.4 1.0 1.0) :Thickness 16)
+                              ;; dot doing idk what, its out there 
+                              (Gizmos.Point :Center (Float3 -2 -0.2 1)  :Color (Float4 1.0 0.2 1.0 1.0) :Thickness 4)))
+
+    ; Draw on top of everything (ignore depth)
+    ;; translation gizmo itself, attached to .translation-0
+    (Gizmos.Context :Queue .editor-queue-no-depth :View .view
+                    :Content (->
+                              .transform-0 (Gizmos.Translation) > .transform-0))
+
+    (Gizmos.Context :Queue .editor-queue-no-depth :View .view
+                    :Content (->
+                              .transform-0 (Gizmos.Rotation) > .transform-0))
```

Comment:
If you put it in the same content as the context above the input will work correctly and only be able to interact with one gizmo at a time